comictaggerlib/pagelisteditor.py

```diff
@@ -57,17 +90,17 @@ class PageListEditor(QtWidgets.QWidget):
     modified = QtCore.pyqtSignal()
 
     pageTypeNames = {
-        PageType.FrontCover: "Front Cover",
-        PageType.InnerCover: "Inner Cover",
-        PageType.Advertisement: "Advertisement",
-        PageType.Roundup: "Roundup",
-        PageType.Story: "Story",
-        PageType.Editorial: "Editorial",
-        PageType.Letters: "Letters",
-        PageType.Preview: "Preview",
-        PageType.BackCover: "Back Cover",
-        PageType.Other: "Other",
-        PageType.Deleted: "Deleted",
+        PageType.FrontCover: "Front Cover (F\U00000332)",
+        PageType.InnerCover: "Inner Cover (I\U00000332)",
+        PageType.Advertisement: "Advertisement (A\U00000332)",
+        PageType.Roundup: "Roundup (R\U00000332)",
+        PageType.Story: "Story (S\U00000332)",
+        PageType.Editorial: "Editorial (E\U00000332)",
+        PageType.Letters: "Letters (L\U00000332)",
+        PageType.Preview: "Preview (P\U00000332)",
+        PageType.BackCover: "Back Cover (B\U00000332)",
+        PageType.Other: "Other (O\U00000332)",
+        PageType.Deleted: "Deleted (X\U00000332)",
```

Comment:
Python files are always utf-8 encoded so `F̲` using the literal value `̲` would be preferred over the Unicode escape

Reply:
It may be that your editor/font does not have enough support for utf-8. The editor you use must support the font and the font being used must support the character.
For example I have PyCharm and leafpad (pre-installed on Fedora KDE) on my current machine and both support `U+0332 ◌̲ COMBINING LOW LINE` however the font Go Mono does not show properly on PyCharm but does on leafpad

![Go Mono-Leafpad](https://user-images.githubusercontent.com/7547075/163031743-63ad1eec-99e7-4905-8c49-688ff9400f67.png)
![Go Mono](https://user-images.githubusercontent.com/7547075/163031745-58b4b1d5-c371-42f3-bfdf-864b782571ef.png)
![DejaVu Sans Mono](https://user-images.githubusercontent.com/7547075/163031731-12bada6b-66ba-4c47-a714-174d54aa11dd.png)
![DejaVu Sans](https://user-images.githubusercontent.com/7547075/163031734-ae00b99a-a3f1-4c5a-b000-cdf1b99ac5f7.png)
![Everson Mono](https://user-images.githubusercontent.com/7547075/163031740-2afd121e-2a1c-4db7-8a74-b7cbed711ba3.png)

Reply:
Using QActions now.